src/dances/tap.js

```diff
@@ -0,0 +1,18 @@
+export default (elem, value, options = {}) => {
+  const options =
```

Comment:
You don't need to do this here. `options = {}` already intialize `options` to an empty object if undefined.
You also have a name conflict